vuepress/docs/next/docs/getting-started/multitenancy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Multitenancy
+
+## Overview
+
+Entando 7.2 introduces multitenancy to the Entando Platform. An Entando Application can be customized to enable a multitenant architecture where tenants share an infrastructure but are securely informationally isolated. This document provides an overview of multitenancy and its Entando implementation.
+
+## Core Concepts
+
+Software multitenacy describes an architecture in which a single software instance runs on a server and serves multiple tenants. Each tenant constitutes a group of users who share specific access privileges to the software instance. A multitenant software application is designed to provide a common architecture while ensuring the segregation of information.
+
+Entando Multitenancy imposes a shared architecture (e.g. the versions of Entando, K8s and Keycloak) while distributing resources (e.g. CPU, memory) across the primary and secondary tenants. The primary tenant is the user group with full access to the default features and functionalities of an Entando instance. Secondary tenants are user groups who share the privileges and capabilities of the primary tenant. 
+
+>Note: In the initial implementation of Entando Multitenancy, an active Component Manager is not available to secondary tenants.
+
+Tenants are informationally isolated from one another with unique data, configuration settings, and user management.
+## Architecture
+
+All tenants rely on a single Entando Application instance for core functionality such as the App Builder, App Engine, and Keycloak. Although not a core feature or installed by default, Redis is necessary for cache management and high availability, which Entando strongly recommends for a multitenant configuration. See **TODO - add tutorial link** to add Redis to your Entando Application.
+
+To isolate its data and configurations, each tenant is identified by its own domain name and allocated its own database schema, Entando Content Deliver Server (CDS) instance, Solr core, and Keycloak client realm. Entando CDS is required to manage static resources external to the Entando App Engine, Solr establishes an external search engine, and the Keycloak realm grants user access.
```

Comment:
First sentence: 'To isolate its data and configurations, each tenant is ...." << this makes it sound as if the unique domain names of each tenant is only for this reason... not to provide the option of having unique domain tenants, like the cause and effect is reversed. Isn't it more like to have a unique domain tenant, we isolate the data and configuration, not the other way round.

Reply:
i don't understand this. x y and z are required to isolate the data and configs of a tenant.

Reply:
rephrased